channels/stable

```diff
@@ -78,7 +78,7 @@ spec:
   - range: ">=1.12.0-alpha.1"
     #recommendedVersion: "1.12.0"
     #requiredVersion: 1.12.0
-    kubernetesVersion: 1.12.7
+    kubernetesVersion: 1.12.8
```

Comment:
We generally hold off on bumping stable right away unless there are security vulnerabilities. Once a few people, attest they've been running it successfully in prod we bump. Do you feel like we can hold off on modifying stable for a bit?

Reply:
As we are not on a stable release yet I don't really see a reason to hold this off much longer tbh. But I can keep this out for now as well